README.md

```diff
@@ -1,2 +1,133 @@
 # Hello-World
 Just getting to know GitHub
+now I am editing on the new readme. this is my code:
+package Objects_Class;
+import java.io.PipedWriter;
+import java.util.ArrayList;
+import java.io.Console;
+import java.util.Scanner;
```

Comment:
these lines of code is redundant: pls check and verify in 1 hour and revert back to me.